### PR TITLE
Register-DbatoolsConfig - Add example that shows -PassThru

### PR DESCRIPTION
**Pull request by jpomfret**
## Please read -- recent changes to our repo
On November 10, 2022, [we removed some bloat from our repository (for the second and final time)](https://github.com/dataplat/dbatools/issues/8542). This change requires that all contributors reclone or refork their repo.

PRs from repos that have not been recently reforked or recloned will be closed and @potatoqualitee will cherry-pick your commits and open a new PR with your changes.

 - [X] Please confirm you have the smaller repo (85MB .git directory vs 275MB or 110MB or 185MB .git directory)

## Type of Change

 - [ ] Bug fix (non-breaking change, fixes # )
 - [ ] New feature (non-breaking change, adds functionality, fixes # )
 - [ ] Breaking change (effects multiple commands or functionality, fixes # )
 - [ ] Ran manual Pester test and has passed (`.\tests\manual.pester.ps1`)
 - [ ] Adding code coverage to existing functionality
 - [ ] Pester test is included
 - [ ] If new file reference added for test, has is been added to github.com/dataplat/appveyor-lab ?
 - [ ] Unit test is included
 - [X] Documentation - #8368 
 - [ ] Build system

### Purpose
Add an example to show the -PassThru parameter of Set-Dbatoolsconfig

### CODE REVIEW

File: public/Register-DbatoolsConfig.ps1 [hunk 1]
```diff
@@ -56,6 +56,13 @@ function Register-DbatoolsConfig {
         PS C:\> Register-DbatoolsConfig -Module Message -Scope SystemMandatory
 
         Retrieves all configuration items of the module Message, then registers them in registry to enforce them for all users on the current system.
+
+    .EXAMPLE
+        PS C:\> Set-DbatoolsConfig -FullName sql.connection.trustcert -Value $true -PassThru | Register-DbatoolsConfig
+
+        Set the `sql.connection.trustcert` configuration to be $true, and then use the -PassThru parameter
```
Comment (wsmelton) [inline public/Register-DbatoolsConfig.ps1:63]: The back ticks don't render code statement on our docs page, that is only for markdown support 😉

Comment (jpomfret) [reply public/Register-DbatoolsConfig.ps1:63]: Gotcha - shall I remove? Or ok just to show separation?

Comment (wsmelton) [reply public/Register-DbatoolsConfig.ps1:63]: I'd remove and put in double-quotes or brackets if you need to call focus to it.

Comment (jpomfret) [reply public/Register-DbatoolsConfig.ps1]: Consider it done :)

**Review by wsmelton — APPROVED** [review]

**Comment by wsmelton** [comment]
Will go ahead and merge this as the test failures don't pertain to doc changes.

**Comment by jpomfret** [comment]
Thanks @wsmelton!

**Pull request merged**